ui/scss/component/_markdown-preview.scss

```diff
@@ -1,4 +1,265 @@
 .markdown-preview {
```

Comment:
Most styles in this file should probably be targeting generic html elements, instead of specifically inside the markdown editor, so that the rest of the app is consistent with the blogs.

I'd also like to avoid multiple different hardcoded padding values (em and px), as well as hard coded colors.